license-ops/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: license-ops
-version: 1.0.0
-appVersion: 1.0.0
+version: 1.0.1
```

Comment:
Everything looks good. Let's only change `appVersion` and leave `version` as is because it is automatically set in our CI scripts.

Reply:
Will fix this soon.